mls-rs-uniffi/src/config.rs

```diff
@@ -53,10 +52,15 @@ impl mls_rs_core::group::GroupStateStorage for ClientGroupStorage {
     }
 }
 
+#[cfg(feature = "rustcrypto")]
+pub(crate) type UniFFICryptoProvider = mls_rs_crypto_rustcrypto::RustCryptoProvider;
+#[cfg(feature = "openssl")]
+pub(crate) type UniFFICryptoProvider = mls_rs_crypto_openssl::OpensslCryptoProvider;
+
```

Comment:
Won't this crash if both features are on? IIUC Rust features should be "additive"?